guestbook/README.md

```diff
@@ -38,6 +54,172 @@ service/redis-slave    ClusterIP   10.108.119.130   <none>        6379/TCP
+% kubectl create namespace guestbook
+namespace/guestbook created
+```
+
+- Apply raw resources in guestbook namespace
+
+```shell
+% kubectl apply -f guestbook/raw -n guestbook
+deployment.apps/frontend created
+service/frontend created
+deployment.apps/redis-master created
+service/redis-master created
+deployment.apps/redis-slave created
+service/redis-slave created
+% kubectl get deploy,svc -n guestbook
+NAME                           READY   UP-TO-DATE   AVAILABLE   AGE
+deployment.apps/frontend       3/3     3            3           21s
+deployment.apps/redis-master   1/1     1            1           21s
+deployment.apps/redis-slave    2/2     2            2           19s
+
+NAME                   TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)        AGE
+service/frontend       NodePort    10.108.42.109   <none>        80:31631/TCP   21s
+service/redis-master   ClusterIP   10.110.64.132   <none>        6379/TCP       20s
+service/redis-slave    ClusterIP   10.99.140.251   <none>        6379/TCP       19s
+```
+
+- Create cluster, cluster namespaces and all-in-one ham operator
+
+```shell
+% kubectl apply -f guestbook/discovery/prerequisites.yaml
+customresourcedefinition.apiextensions.k8s.io/clusters.clusterregistry.k8s.io created
+namespace/toronto created
+cluster.clusterregistry.k8s.io/toronto created
+operator.deploy.hybridapp.io/ham-sample created
+% kubectl get clusters --all-namespaces
+NAMESPACE   NAME      AGE
+toronto     toronto   1m
+% kubectl get operator,pod
+NAME                                      AGE
+operator.deploy.hybridapp.io/ham-sample   125m
+
+NAME                                          READY   STATUS    RESTARTS   AGE
+pod/ham-sample-pod                            3/3     Running   0          3m55s
+```
+
+#### Discovery
+
+- Apply Application for discovery tool
+
+``` shell
+% kubectl apply -f guestbook/discovery/model.yaml
+application.app.k8s.io/gbapp created
+% kubectl get deployables.apps.open-cluster-management.io -n toronto
+NAME                                                         TEMPLATE-KIND   TEMPLATE-APIVERSION          AGE    STATUS
+deployable-default-deployment-guestbook-frontend-crgxc       Deployable      core.hybridapp.io/v1alpha1   5s     
+deployable-default-deployment-guestbook-redis-master-22m5f   Deployable      core.hybridapp.io/v1alpha1   5s     
+deployable-default-deployment-guestbook-redis-slave-gw6t8    Deployable      core.hybridapp.io/v1alpha1   4s     
+deployable-default-service-guestbook-frontend-44xjn          Deployable      core.hybridapp.io/v1alpha1   4s     
+deployable-default-service-guestbook-redis-master-hc75w      Deployable      core.hybridapp.io/v1alpha1   4s     
+deployable-default-service-guestbook-redis-slave-5wtpz       Deployable      core.hybridapp.io/v1alpha1   4s     
+gbapp-tjzfl                                                  Application     app.k8s.io/v1beta1           104s 
+```
+
+- Trigger resource generation and assembly, uncomment the tool
```

Comment:
I think we need to be specific about which annotation needs to be uncommented, `tools.hybridapp.io/hybrid-discovery-create-assembler` in our case, as there can be more then 1 tools annotation , `tools.hybridapp.io/hybrid-discovery-clusterscoped` being another one. A simple sed command which will uncomment the annotation looks like this : `sed -i -e '/hybrid-discovery-create-assembler/s/#//' guestbook/discovery/model.yaml`